service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/LagomConfigComponent.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.api
+
+import com.typesafe.config.Config
+import play.api.Configuration
+
+trait LagomConfigComponent {
+  def configuration: Configuration
+  def config: Config = configuration.underlying
+}
```

Comment:
I had to add this extra component because we had a couple of components defining 
```scala
def config: Config = configuration.underlying
```
When they get mixed together, we get a compilation error.

This is currently defined in `scaladsl.api`, but ideally we should have another thin module with general purpose components that could be used by the `persistence` module as well (see comment [above](https://github.com/lagom/lagom/pull/881/files#r128861287)).